test/fixtures.rb

```diff
@@ -255,3 +255,12 @@ class PureValidator
 
   validates :name, presence: true
 end
+
+class UserDefinedValidationTest
+  include Lotus::Validations
+  attr_accessor :foo
+
+  validates :foo, :is_bar do |attribute|
```

Comment:
I actually like that idea as well. In my apps I sometimes use different Error classes and pattern match over them. This makes it easier to serialize certain types of errors. So yes, adding your own error class seems wise to me.